optimum/exporters/onnx/__main__.py

```diff
@@ -116,7 +116,12 @@ def main():
         args.for_ort and (model.config.is_encoder_decoder or task.startswith("causal-lm"))
     ):
         if task == "stable-diffusion":
-            output_names = ["text_encoder/model.onnx", "unet/model.onnx", "vae_decoder/model.onnx"]
+            output_names = [
+                "text_encoder/model.onnx",
+                "unet/model.onnx",
+                "vae_encoder/model.onnx",
+                "vae_decoder/model.onnx",
```

Comment:
Aren't we duplicating memory by doing this? I think we should definitely avoid it.

Reply:
Actually not, the encoder and decoder are obviously separate. My bad.